shared/src/business/useCaseHelper/pdf/parseAndScrapePdfContents.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Parse and Scrape PDF Contents
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext the application context
+ * @param {string} providers.pdfBuffer the pdf buffer
+ * @returns {object} the parsed and scraped pdf contents
+ */
+exports.parseAndScrapePdfContents = async ({
+  applicationContext,
+  pdfBuffer,
+}) => {
+  const arrayBuffer = new ArrayBuffer(pdfBuffer.length);
+  const view = new Uint8Array(arrayBuffer);
+  for (let i = 0; i < pdfBuffer.length; ++i) {
```

Comment:
Is this skipping the 0th index?

Reply:
I'm also wondering if the same goal to transform a Buffer to a Uint8Array could be achieved with something like this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/from

Reply:
Would it be okay if we add a card to the refactoring backlog to review this? We know it's working, and it's code that was already there that was moved into a separate file for reuse.

Reply:
sure! on second look, I'm not even sure this `view` constant is being used right now? Either way, that's fine.

Reply:
Added to trello: https://trello.com/c/pHPuSKfL/853-review-the-strange-code-in-parseandscrapepdfcontents